docs/how_to_submit_a_pull_request_on_github.md

```diff
@@ -1,5 +1,59 @@
-:construction: UNDER CONSTRUCTION :construction:
+How to Submit a WSO Pull Request on Github
+==========================================
 
-TODO:
+According to [Rene](http://www.agileventures.org/users/rene-paulokat), the code review process in WSO is something [like this video](https://youtu.be/QEN5-_93gQg)
 
- * [ ] pull in contents from http://www.agileventures.org/projects/websiteone/documents/how-to-submit-a-pull-request-on-github
\ No newline at end of file
+Assuming that you have the following local setup:
+
+```
+$ git remote -v
+origin	   git@github.com:YourName/WebsiteOne.git (fetch)
+origin	   git@github.com:YourName/WebsiteOne.git (push)
+upstream   https://github.com/AgileVentures/WebsiteOne.git (fetch)
+upstream   https://github.com/AgileVentures/WebsiteOne.git (push)
+```
+1) First, make sure you are checked in to the right branch:
+
+```
+$ git checkout <your_feature_branch>
+```
+
+2) Pull the most recent version of 'develop' from the WSO GitHub repo with
+
+```
+$ git pull upstream develop
+```
+
+This fetches and merges your current branch with the latest upstream/develop branch. If any conflicts need to be resolved at this step, how you do so will depend on what tools/IDE you use locally. This goes beyond a quick explanation - pair with someone if you get stuck.
+
+3) Run all tests.
+
+4) Push the feature branch up to your GitHub repo with
+
+```
+$ git push origin <your_feature_branch>
+```
+
+5) When you view this branch on GitHub, click the green button whose tooltip says 'Compare, review, create pull request':
+
+![Compare, review, create pull request](https://dl.dropboxusercontent.com/u/20922989/Screen%20Shot%202014-08-09%20at%2012.36.44%20AM.png)
+
+Or, if you visit your repo shortly after pushing the branch, there will be a prominent notice about the push, with a prompt to create a pull request:
+
+![Compare and pull request](https://dl.dropboxusercontent.com/u/20922989/Screen%20Shot%202014-08-09%20at%2011.25.32%20AM.png)
+
+6) Fill out the description with a link to the Pivotal Tracker story & a brief description of what you've done and create the pull request.
```

Comment:
Not necessary with waffle but perhaps something about naming convention and comment convention on the issue/PR to ensure waffle picks it up correctly?